graphs/deep_clone_graph.py

```diff
@@ -0,0 +1,61 @@
+"""
+LeetCode 133. Clone Graph
+https://leetcode.com/problems/clone-graph/
+
+Given a reference of a node in a connected undirected graph.
+
+Return a deep copy (clone) of the graph.
+
+Each node in the graph contains a value (int) and a list (List[Node]) of its
+neighbors.
+"""
+
+
+class Node:
+    def __init__(self, value=0, neighbors=None):
```

Comment:
```suggestion
    def __init__(self, value: int = 0, neighbors: list | None = None) -> None:
```

Or better yet, make Node a dataclass.